hi-ml-azure/run_pytest.py

```diff
@@ -46,6 +53,10 @@ class RunPytestConfig(param.Parameterized):
         default="30m", doc="The maximum runtime that is allowed for this job in AzureML. This is given as a floating"
         "point number with a string suffix s, m, h, d for seconds, minutes, hours, day. Examples: '3.5h', '2d'"
     )
+    sys_path: str = param.String(
```

Comment:
I think the name of this suggests that it will replace the entire sys.path rather than be appended to it?